<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PushWindow</class>
 <widget class="QDialog" name="PushWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PushWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 20px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
   </property>
   <property name="text">
    <string>Generate commands</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="commit_input">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 20px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="commit_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 15px;
border: 7 solid black;
font: 75 13pt &quot;Dyuthi&quot;;</string>
   </property>
   <property name="text">
    <string>   Enter commit</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 20px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="commit_label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 15px;
border: 7 solid black;
font: 75 13pt &quot;Dyuthi&quot;;</string>
   </property>
   <property name="text">
    <string>  Enter your email</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="email_input">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 20px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="commit_label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 15px;
border: 7 solid black;
font: 75 13pt &quot;Dyuthi&quot;;</string>
   </property>
   <property name="text">
    <string>  Url to repository</string>
   </property>
  </widget>
  <widget class="QLabel" name="token_label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 15px;
border: 7 solid black;
font: 75 13pt &quot;Dyuthi&quot;;</string>
   </property>
   <property name="text">
    <string>  Enter your name</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="url_input">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 20px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
